internal/api/annotations/annotations.go

```diff
@@ -1,8 +1,29 @@
+/*
+Copyright 2024.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
 package annotations
 
 const (
 	SELinuxPolicyConfigAnnotation = "config.node.openshift-kni.io/selinux-policy"
 	SELinuxPolicyCustom           = "custom"
+	// MultiplePoolsPerTreeAnnotation an annotation used to re-enable the support of multiple node pools per tree; starting 4.18 it is disabled by default
+	// allowed values are "enabled" or "disabled"
+	MultiplePoolsPerTreeAnnotation = "config.node.openshift-kni.io/multiple-pools-per-tree"
+	MultiplePoolsPerTreeEnabled    = "enabled"
+	MultiplePoolsPerTreeDisabled   = "disabled"
```

Comment:
is "disabled" ever used?

Reply:
never out of tests. basically any value except for "enabled" is considered "disabled", I'm in favor to remove it completely

Reply:
considering the magic annotation should never be documented except for selected controlled cases, yes, let's remove it to minimize the surface.